### ISSUE TITLE
--gen-pkg-only should print the path to the resulting package

### ISSUE THREAD

**Issue by ssokolow** [fornwall/rust-script#17]
Currently, it's a major hassle to identify the output directory for `--gen-pkg-only` if I want to build a command for "OK, this 'one-off' script has outgrown a single file. Let's turn it into a full project."

`--gen-pkg-only` should print the path it's writing to so I can do something like this

```sh
outdir="$(rust-script --gen-pkg-only "$1")"
mv "$1" "$2"
cd "$2"
git init
git commit -a -m "Unpack '$1' into its own project"
```

Maybe also a flag that does similar for the generated binary to streamline the "I want Rust for its cold-cache startup performance. Bypass Cargo entirely" case where you'd otherwise have to use `--cargo-output` and then copy-paste the path out of it.

**Comment by SafariMonkey** [comment]
@ssokolow  Does the `--pkg-path` option help you?
```
        --pkg-path <pkg_path>
            Specify where to place the generated Cargo package.
```
So your script would just be
```
rust-script --gen-pkg-only "$1" --pkg-path "$2"
cd "$2"
git init
git commit -a -m "Unpack '$1' into its own project"
```
(assuming `mv "$1" "$2"` in your script was meant to be `mv "$outdir" "$2"`)

I still want to implement this (in fact, I already have) but I think that makes more sense for the workflow you outlined.

**Comment by ssokolow** [comment]
I'm big on minimizing time spent waiting on the tooling in my efforts to replace Python with Rust for my "quick hacks that often turn out to not be so disposable"... to the point that, for using full-blown packages that way, I've been exploring the idea of something akin to cargo-generate which also produces a `target` directory pre-populated with the results of running `cargo build` on the template with a placeholder package name.

...and that means I'd really prefer to just grab what I'd already been iterating on under rust-script with something as close as possible to `mv rust_script_pkg some_directory_on_the_same_partition`.

**Comment by SafariMonkey** [comment]
Here is my commit: https://github.com/fornwall/rust-script/commit/0b97d86b1e0897cd6ba2e26bc90852ae0ad82bc3. I want to add another format for IDE-relevant output, but that should be enough for what you're looking for. Please let me know if you have any thoughts on it.

I have a number of local changes from lint fixes to the whole IDE integration, so I don't really want to try putting out pull requests just yet.

**Comment by fornwall** [comment]
In the just released [version 0.23.0](https://github.com/fornwall/rust-script/releases/tag/0.23.0) the `-p`/`--package` (which `--gen-pkg-only` has been renamed to) option prints the path to the package, allowing flexibility as in `cargo tree --manifest-path $(rust-script -p test.rs)/Cargo.toml`.

Let me know how it works!
